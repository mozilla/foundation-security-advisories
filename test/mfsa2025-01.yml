# Basic testcases
announced: January 1, 2025
impact: low
fixed_in:
  - Firefox 128
  - Firefox ESR 115.1
title: Test Security Vulnerabilities fixed in Firefox 128 and Firefox ESR 115.1
advisories:
  MFSA-TMP-2025-00001:
    title: Test advisory with title and description
    impact: low
    reporter: Reporter
    description: |
      Description.
    bugs:
      - url: 1000000
  MFSA-TMP-2025-00002:
    title: Test advisory only with title
    impact: low
    reporter: Reporter
    bugs:
      - url: 1000000
