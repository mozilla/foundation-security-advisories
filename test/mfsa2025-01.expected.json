{
  "MFSA-TMP-2025-00001": {
    "containers": {
      "cna": {
        "affected": [
          {
            "product": "Firefox",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "128",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          },
          {
            "product": "Firefox ESR",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "115.1",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          }
        ],
        "credits": [
          {
            "lang": "en",
            "value": "Reporter"
          }
        ],
        "descriptions": [
          {
            "lang": "en",
            "supportingMedia": [
              {
                "base64": false,
                "type": "text/html",
                "value": "Description. This vulnerability affects Firefox < 128 and Firefox ESR < 115.1."
              }
            ],
            "value": "Description. This vulnerability affects Firefox < 128 and Firefox ESR < 115.1."
          }
        ],
        "references": [
          {
            "url": "https://www.mozilla.org/security/advisories/mfsa2025-01/"
          },
          {
            "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1000000"
          }
        ],
        "title": "Test advisory with title and description"
      }
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
  },
  "MFSA-TMP-2025-00002": {
    "containers": {
      "cna": {
        "affected": [
          {
            "product": "Firefox",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "128",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          },
          {
            "product": "Firefox ESR",
            "vendor": "Mozilla",
            "versions": [
              {
                "lessThan": "115.1",
                "status": "affected",
                "version": "unspecified",
                "versionType": "custom"
              }
            ]
          }
        ],
        "credits": [
          {
            "lang": "en",
            "value": "Reporter"
          }
        ],
        "descriptions": [
          {
            "lang": "en",
            "supportingMedia": [
              {
                "base64": false,
                "type": "text/html",
                "value": "Test advisory only with title. This vulnerability affects Firefox < 128 and Firefox ESR < 115.1."
              }
            ],
            "value": "Test advisory only with title. This vulnerability affects Firefox < 128 and Firefox ESR < 115.1."
          }
        ],
        "references": [
          {
            "url": "https://www.mozilla.org/security/advisories/mfsa2025-01/"
          },
          {
            "url": "https://bugzilla.mozilla.org/show_bug.cgi?id=1000000"
          }
        ],
        "title": "Test advisory only with title"
      }
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
  }
}